src/templates/default_nativeapp/app/setup_script.sql

```diff
@@ -0,0 +1,48 @@
+CREATE OR REPLACE APPLICATION ROLE app_instance_role;
+CREATE OR REPLACE SCHEMA non_versioned_schema;
+
+    GRANT USAGE ON SCHEMA non_versioned_schema
+        TO APPLICATION ROLE app_instance_role;
+
+    CREATE OR REPLACE
+        TABLE non_versioned_schema.id_table(id INT);
+
+    INSERT INTO
+        non_versioned_schema.id_table VALUES (1);
+
+    GRANT SELECT
+        ON non_versioned_schema.id_table
+        TO APPLICATION ROLE app_instance_role;
+
+
+CREATE OR ALTER VERSIONED SCHEMA versioned_schema;
+
+    CREATE OR REPLACE
```

Comment:
Is there any reason for different indentation?

Reply:
It is for ease of reading. Indentation exists for objects that belong to the same schema.